src/main/java/com/synopsys/integration/alert/channel/jira/server/web/JiraServerCustomFunctionAction.java

```diff
@@ -76,15 +76,15 @@ public JiraServerCustomFunctionAction(AuthorizationManager authorizationManager,
             String username = jiraProperties.getUsername();
             String password = jiraProperties.getPassword();
             try {
-                jiraAppService.installMarketplaceServerApp(JiraConstants.JIRA_APP_KEY, username, password);
+                jiraAppService.installMarketplaceServerApp(JiraConstants.JIRA_APP_KEY);
             } catch (IntegrationRestException e) {
                 if (RestConstants.NOT_FOUND_404 == e.getHttpStatusCode()) {
                     return new ActionResponse<>(HttpStatus.NOT_FOUND,
                         "The marketplace listing of the Alert Issue Property Indexer app may not support your version of Jira. Please install the app manually or request a compatibility update. Error: " + e.getMessage());
                 }
                 createBadRequestIntegrationException(e);
             }
-            boolean jiraPluginInstalled = JiraPluginCheckUtil.checkIsAppInstalledAndRetryIfNecessary(jiraAppService, username, password);
+            boolean jiraPluginInstalled = JiraPluginCheckUtil.checkIsAppInstalledAndRetryIfNecessary(jiraAppService);
             if (!jiraPluginInstalled) {
                 return new ActionResponse<>(HttpStatus.NOT_FOUND, "Was not able to confirm Jira server successfully installed the Jira Server plugin. Please verify the installation on you Jira server.");
```

Comment:
Same issue as my previous comment. We probably want to update this message with the name of the App so they know what to look for in the installed apps, and if it isn't there then they know what to search for in the marketplace.